### ISSUE TITLE
Log Most Messages in Harvest to the Harvest Log instead of the UI

### ISSUE THREAD

**Issue by acouch** [GetDKAN/dkan#2819]
Currently if there is an error with the resource creation in Harvest ``displayMessage()`` tells the user. This ends up with a lot of information that most users of the harvest are not able to act upon. Additionally these items are not logged for future reference when they could be most useful. 

Example:

![image](https://user-images.githubusercontent.com/512243/52653311-bd3b7d80-2ebd-11e9-9baf-e3cd55079b74.png)

Here is a link in the code: https://github.com/GetDKAN/dkan/blob/7.x-1.x/modules/dkan/dkan_harvest/dkan_harvest.migrate.inc#L1105

These messages should be routed to the harvest error and event log.
**error examples**: 
- Resource remote url (https://data.1111/download/filename.accdb) extension (accdb) not allowed
- Resource remote url (http://) not valid!

**event examples**
- Groups were updated on 3 resource(s).
- This filemime type (application/pdf) can be added as a resource, but cannot be imported to our datastore.
- Node 1042691 doesn't have a proper file path.
- 26 resources were queued to be updated with their new groups. They will be processed during upcoming cron runs.
- Processed 5 (0 created, 5 updated, 0 failed, 0 ignored) in 19.4 sec (15/min) - done with 'dkan_harvest_migrate_example'

I'm not sure how the the logging in the code is wired up so that will need a little more discovery.

### Acceptance Criteria

- [ ] Running a harvest using this file https://s3.amazonaws.com/dkan-default-content-files/ca-dkan-demo-data.json should not produce errors in the user interface for each resource
- [ ] Running the above should save the errors to the error log (remove identical repeated errors)
- [ ] Informational messages should be saved to the event log (remove identical repeated messages)
- [ ] There should be a summary displayed to the user if there harvest has zero errors ("This harvest ran error-free")
- [ ] There should be a summary displayed to the user if there are errors ("This harvest ran had some errors. Please see the log for details.")
- [ ] Suppress/Remove Acquia Purge errors (i.e. The system cannot publicly refresh the changes you just made, because of the following error conditions...)

blocked by #2842

**Comment by kimwdavidson** [comment]
Adding information about extensions from #2460:

To replicate:

1. Create a new harvest for https://open.fda.gov/data.json. Click through to "Harvest Now".
2. Once the harvest has completed, note that there are a large number of errors showing on the page: things like "Resource remote url (http://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfRES/res.cfm) extension (cfm) not allowed" and "Resource remote url (http://) not valid!" in pink. There are also a large number of messages in green, but many of them seem to qualify as errors of some sort: "Node 43 doesn't have a proper file path." or "This filemime type (application/zip) can be added as a resource, but cannot be imported to our datastore."
3. Visit the harvest's Errors page (example: http://demo.getdkan.com/node/41/harvest-errors). There will be no errors listed.

**Comment by janette** [comment]
![harvest-messages](https://user-images.githubusercontent.com/314172/54049312-c6a8c480-41a1-11e9-8c87-64133e58f22b.png)

**Comment by dharizza** [comment]
The errors were already logged in harvest UI. 
The green messages are still loaded during and after the harvest but they can't be moved to a specific harvest tab as it is not possible to relate them to the harvest. Also, the "Events" tab is meant to store data about every time a harvest is run, not the status messages related to it. 

About the messages shown, some of them are related to nodes (like "Node 43 doesn't have a proper file path.") but some don't have an ID so nothing is actionable from "This filemime type (application/zip) can be added as a resource, but cannot be imported to our datastore." or "Groups were updated on 1 resource(s).", according to this and the fact that the AC on this ticket doesn't apply anymore to the current functionality, we decided to just hide the messages "This filemime type (application/zip) can be added as a resource, but cannot be imported to our datastore." and "Groups were updated on 1 resource(s)." from status page when harvest is being run and from the dkan harvest dashboard page. The work for this is done here: https://github.com/GetDKAN/dkan/pull/2903
